pyserini/convert_to_tokenized_json.py

```diff
@@ -0,0 +1,45 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from transformers import BertTokenizer, T5Tokenizer
+import argparse
+import json
+
+def main(args):
+    if (args.tokenizer == 'bert'):
+        tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
```

Comment:
okay to assume user specifies fully-qualified name, e.g.,
```
tokenizer = BertTokenizer.from_pretrained(args.tokenizer)
```

But make arg optional, e.g., default to bert-base-uncased?

Reply:
Yes, I will make it optional. 
But if we want t5 tokenizer, it is called from `**T5Tokenizer.**from_pretrained('castorini/doc2query-t5-base-msmarco')`. still cannot get rid of if statement?

Reply:
ah, I see... maybe like a comprise and search for a substring of {bert, t5} in the if?